docs/content/routing/entrypoints.md

```diff
@@ -194,6 +194,62 @@ If both TCP and UDP are wanted for the same port, two entryPoints definitions ar
     --entryPoints.udpep.address=:3179/udp
     ```
 
+#### listen on some ip addresses only
+
+```toml tab="File (TOML)"
+[entryPoints.oneIPv4AddressEp]
+  address = "192.168.2.7:8888"
+[entryPoints.oneIPv6AddressEp]
+  address = "[2001:db8::1]:8888"
+```
+
+```yaml tab="File (yaml)"
+entryPoints:
+  oneIPv4AddressEp:
+    address: "192.168.2.7:8888"
+  oneIPv6addressEp:
+    address: "[2001:db8::1]:8888"
+```
+
+```bash tab="CLI"
+entrypoints.oneIPv4AddressEp.address=192.168.2.7:8888
+entrypoints.oneIPv6AddressEp.address=[2001:db8::1]:8888
+```
+
+Full details for how to specify `address` can be found in [net.Listen](https://golang.org/pkg/net/#Listen) (and [net.Dial](https://golang.org/pkg/net/#Dial)) of the doc for go.
+
+!!! warning
+
+    If using an orchestrator, the IP addresses available to bind to may not be externally accessible.
+
+??? info "Docker"
+
+    Publish the ports only on the desired addresses. The ip address of the host is not visible.
```

Comment:
this example is outside of the context of configuring entrypoints, it's unclear what the relation of this example has with the examples above it.

recommend removing it or adding relevant examples to the entrypoint configuration, in addition to explaining the context switch, in addition to adding a comment to the yaml indicating the reader is looking at a compose template